core/services/relay/evm/evmtesting/chain_components_interface_tester.go

```diff
@@ -489,3 +515,25 @@ func MidStaticToInternalType(m MidLevelStaticTestStruct) chain_reader_tester.Mid
 		},
 	}
 }
+
+// injectAddressModifier injects the EVMAddressModifier into the ChainReaderConfig after decoding.
+// This is necessary because the AddressModifier interface is not serializable and must be manually
+// set after the configuration is deserialized from JSON
+func injectAddressModifier(conf *types.ChainReaderConfig) {
```

Comment:
How would this work when run without a tester? The instance should be injected during the ChainReader service runtime